setup/windows setup.md

```diff
@@ -0,0 +1,16 @@
+Installing the JDK
+# Windows Setup
+## Java JDK Installation
+1. Download the JDK installer for Windows from the Oracle website: 
+http://www.oracle.com/technetwork/java/javase/downloads/index.html
+2. Run the installer
+3. Add the bin\ directory of the installed JDK to the PATH environment variable, as described here: http://www.java.com/en/download/help/path.xml
```

Comment:
Could you put backticks around `bin\` and `PATH` here to indicate that they need to be input by the user? (It will show them in a monospace font).

Could you do the same thing on the remaining lines for these?
- `java -version`
- `sbt about`
- `[info] This is sbt 0.13.x`